lib/routes/charge/sale_view.dart

```diff
@@ -70,31 +75,98 @@ class SaleViewState extends State<SaleView> {
                     iconTheme: Theme.of(context).appBarTheme.iconTheme,
                     textTheme: Theme.of(context).appBarTheme.textTheme,
                     backgroundColor: Theme.of(context).canvasColor,
-                    leading: backBtn.BackButton(iconData: Icons.close),
-                    title: _TotalSaleCharge(
-                      accountModel: accModel,
-                      currentSale: currentSale,
-                      useFiat: widget.useFiat,
-                    ),
+                    leading: backBtn.BackButton(),
+                    title: Text("Current Sale"),
                     actions: <Widget>[
                       IconButton(
                         icon: Icon(
-                          Icons.note_add,
+                          Icons.delete_forever,
                           color: Theme.of(context).iconTheme.color,
                         ),
-                        onPressed: () {},
+                        onPressed: () {
+                          widget.onDeleteSale();
+                        },
                       )
                     ],
                     elevation: 0.0,
                   ),
-                  body: SaleLinesList(
-                      accountModel: accModel, currentSale: currentSale),
-                  bottomNavigationBar: SingleButtonBottomBar(
-                    text: "Clear Sale",
-                    onPressed: () {
-                      posCatalogBloc.actionsSink
-                          .add(SetCurrentSale(Sale(saleLines: [])));
-                    },
+                  extendBody: false,
+                  backgroundColor: Theme.of(context).backgroundColor,
+                  body: Stack(
+                    fit: StackFit.expand,
+                    children: [
+                      Column(
+                        mainAxisSize: MainAxisSize.min,
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        children: [
+                          Container(
```

Comment:
Please remove commented out "Add Note" widget from pos_invoice if this is final.

Reply:
Done